test/Microsoft.Health.Fhir.Shared.Tests.Crucible/CrucibleTestFixture.cs

```diff
@@ -37,12 +40,14 @@ public void Run(string id)
 
             if (findTest != null)
             {
+                bool isCosmosDb = CrucibleDataSource.GetDataStore().Equals(DataStore.CosmosDb);
```

Comment:
In general we prefer to use string.Equals and also pass in the type of string comparison, so for example here: `string.Equals(a, b, StringComparison.OrdinalIgnoreCase)`